include/libdnf/module/module_dependencies.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_MODULE_MODULE_DEPENDENCIES_HPP
+#define LIBDNF_MODULE_MODULE_DEPENDENCIES_HPP
+
+#include <map>
+#include <string>
+#include <vector>
+
+struct _ModulemdDependencies;
+
+
+namespace libdnf::module {
+
+
+class ModuleDependencies {
```

Comment:
Do we really need that class? If yes do we really need to store inside `ModulemdDependencies * dependencies`? We can replace it by a `vector` with `map`

Reply:
Changed according to the suggestion below.